quiz.js

```diff
@@ -0,0 +1,100 @@
+//CarLot augmentor iife for event handlers
+window.onload = function () {
+  CarLot.loadInventory(CarLot.populatePage);
+};
+
+var CarLot = (function (oldCarLot) {
+  oldCarLot.populatePage = function (inventoryPH) {
+
+    var container = document.getElementById("divLord");
+
+    for (var j=0; j<inventoryPH.length; j++) {
+      container.innerHTML += "<div class='col-xs-3 card'><div class='cardInfo'></div></div>"
+    }
+
+    //logic should be the same below here
+    var cards = document.querySelectorAll("div.cardInfo");
+    for (var i=0; i<inventoryPH.length; i++) {
+      cards[i].innerHTML +=
+       "<span class='titleSpan'>Manufacturer: </span><span>"
+       +  inventoryPH[i].make
+       + "</span><br><span class='titleSpan'>Model: </span><span>"
+       + inventoryPH[i].model
+       + "</span><br><span class='titleSpan'>Year: </span><span>"
+       + inventoryPH[i].year
+       + "</span><br><span class='titleSpan'>Price: </span><span>"
+       + inventoryPH[i].price
+       + "</span><br><span class='titleSpan'>Color: </span><span>"
+       + inventoryPH[i].color
+       + "</span><br><span class='titleSpan'>Purchased: </span><span>"
+       + inventoryPH[i].purchased
+       + "</span><br><span class='titleSpan'>Description: </span><span class='desc'>"
+       + inventoryPH[i].description
+       + "</span";
+
+        cards[i].style.borderColor = inventoryPH[i].color;
+    }
+    console.log("populate page elements done")
+    CarLot.activateEvents();
+  }
+
+  oldCarLot.activateEvents = function () {
+    var divFocus;
+    document.getElementById("submitButton").addEventListener("click", function(event){
+        event.preventDefault()
+    });
+
+
+    var cards = document.querySelectorAll("div.cardInfo");
+
+    for (var i=0; i<cards.length; i++) {
+      cards[i].addEventListener("click",function(){
+        console.log("click");
+        divFocus = this;
+        document.getElementById("searchBar").placeholder = "";
```

Comment:
You should have some functionality in here that resets the cards as you can select multiple cards.